op-e2e/op_geth_test.go

```diff
@@ -44,9 +44,17 @@ func TestMissingGasLimit(t *testing.T) {
 // TestTxGasSameAsBlockGasLimit tests that op-geth rejects transactions that attempt to use the full block gas limit.
 // The L1 Info deposit always takes gas so the effective gas limit is lower than the full block gas limit.
 func TestTxGasSameAsBlockGasLimit(t *testing.T) {
+	if externalL2Nodes != "" {
```

Comment:
I've considered a bit how we might have clients advertise what test classes they should support.  My first inclination would be to have the execution of the `--init` step write some sort of compatibility info (probably a JSON file?).  The `TestMain` could consume that compatibility info and set package variables to selectively skip incompatible tests.

For the moment, as suggested, I'd like to ignore this portion of the problem, but wanted to mention the above as a possible future path forward.